docs/_ldio/ldio-outputs/ldio-repository-materialiser.md

```diff
@@ -13,8 +13,20 @@ Any triplestore that supports the RDF4J remote repository API can be used.
 
 ## Config
 
-| Property      | Description                                  | Required | Default | Example                 | Supported values          |
-|:--------------|:---------------------------------------------|:---------|:--------|:------------------------|:--------------------------|
-| sparql-host   | The url of the server hosting the repository | Yes      | N/A     | http://repositoryServer | URL                       |
-| repository-id | The rdf4j repository id                      | Yes      | N/A     | repoId                  | String                    |
-| named-graph   | The timestamp path used for naming the       | No       | N/A     | http://name             | Any valid LD subject name |
+| Property      | Description                                                                                                 | Required | Default | Example                 | Supported values |
+|:--------------|:------------------------------------------------------------------------------------------------------------|:---------|:--------|:------------------------|:-----------------|
+| sparql-host   | The url of the server hosting the repository                                                                | Yes      | N/A     | http://repositoryServer | URL              |
+| repository-id | The rdf4j repository id                                                                                     | Yes      | N/A     | repoId                  | String           |
+| named-graph   | If set, the triples will be written to this named graph                                                     | No       | N/A     | http://name             | Any valid IRI    |
+| batch-size    | Number of members or models that will be committed at once                                                  | No       | 10000   | 500                     | Integer          |
+| batch-timeout | If the batch size has not reached in this amount of milliseconds yet, the members will be committed anyway. | No       | 120000  | 30000                   | Integer          |
```

Comment:
If the batch size has not reached in this amount of milliseconds yet -> If the batch size has not been reached within this number of milliseconds.